src/components/Modal/Modal.jsx

```diff
@@ -163,6 +185,9 @@ Modal.defaultProps = {
   wide: false,
   previousFocus: null,
   dismissible: true,
+  customWidth: null,
```

Comment:
Do you think we can add a `width` props instead?
We can make it accept both a custom size, and a preset size `['wide, …]`.
This will coexist with the `wide` prop for some time and eventually, we will deprecate `wide`.